testing/sdk_networks.py

```diff
@@ -44,17 +48,29 @@ def check_task_network(task_name, expected_network_name="dcos"):
                     )
 
 
+@retrying.retry(
+    wait_fixed=1000, stop_max_delay=5 * 1000, retry_on_result=lambda res: not res
+)
+def wait_for_endpoint_info(
+    package_name: str, service_name: str, endpoint_name: str, json: bool = True
+) -> typing.Dict:
```

Comment:
The addition of the `json` arg means that this return type is no longer valid. Is there a better way to handle this than overloading the function?

Reply:
For now, I could set the return type to `Union[Dict, str]`. Unfortunately not every service endpoint is json serializable at the moment.  Fixing this behaviour would probably include changing existing endpoint definition of services.